wordpress-mobile/gutenberg-mobile
# Image Size Options UI - Add Appium Tests

Resolves part of  #1329 

For enabling removal of DEV flag, see connecting size options UI to networking for updating image urls and minor enhancements here: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1452 and https://github.com/wordpress-mobile/gutenberg-mobile/pull/1453

**More info:**
Gutenberg side PR: https://github.com/WordPress/gutenberg/pull/17245


![image-options-android](https://user-images.githubusercontent.com/1103838/66963443-8e673f80-f028-11e9-84bb-7f9249aae59a.gif)
![ios-size-options](https://user-images.githubusercontent.com/1103838/66963451-932bf380-f028-11e9-9508-8708a10d6164.png)


To test:

1. Build Example app
2. Click on an Image block (that already has an image)
3. Press the gear button to bring up Image Options menu
4. Verify new Size option exists, click size options
5. Verify size options are shown and choose one.
6. Re-open Image options menu and verify previously selected size is now shown as selected.

For verifying sizeSlug is correctly submitted to web backend:

1. Run the metro bundler for this branch and build either Android or iOS WordPress app
2. Use a wordpress.com account and navigate to a post, and begin editing in Gutenberg editor.
3. Follow steps above to update Image Size options. Click Update.
4. One the web, log in to the wordpress.com account, navigate to the article, and verify that the new Image size sizeSlug  setting is shown.


Update release notes (hidden behind __DEV__ flag until networking for updating image urls is completed):
